<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DropN900 - Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
	background-color: white;
	color: black;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>991</width>
        <height>963</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="main_layout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="downloading_icon">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>48</width>
               <height>84</height>
              </size>
             </property>
             <property name="styleSheet">
              <string>padding-bottom: 2px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="title_download_folder">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string>QLabel#title_download_folder, #title_authentication, #title_sync {
	color: #0099FF;
	font-size: 20pt;
	padding-top: 40px;
	padding-bottom: 10px;
}</string>
             </property>
             <property name="text">
              <string>Downloading</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>1</width>
                 <height>1</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QFrame" name="frame">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>26</height>
                </size>
               </property>
               <property name="styleSheet">
                <string>QFrame {
	border: 0px;
	border-top: 2px solid #0099FF;
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string>QLabel {
	color: white;
	font-size: 19pt;
}</string>
           </property>
           <property name="text">
            <string>Default Download Folder</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="download_layout">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLineEdit" name="lineedit_default_download_folder">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>65</height>
              </size>
             </property>
             <property name="styleSheet">
              <string>QLineEdit {
	border: 1px solid gray;
	border-radius: 7px;
	padding: 5px;
	font-size: 19pt;
}</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_browse_folder">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>65</height>
              </size>
             </property>
             <property name="styleSheet">
              <string/>
             </property>
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkbox_no_dl_dialog">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>65</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>QCheckBox {
	color: white;
	font-size: 19pt;
}

QCheckBox::disabled {
	color: grey;
}</string>
           </property>
           <property name="text">
            <string>Don't show dialog on download, always use default folder</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="authentication_icon">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>48</width>
               <height>84</height>
              </size>
             </property>
             <property name="styleSheet">
              <string>padding-bottom: 2px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="title_authentication">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string>QLabel#title_download_folder, #title_authentication, #title_sync {
	color: #0099FF;
	font-size: 20pt;
	padding-top: 40px;
	padding-bottom: 10px;
}</string>
             </property>
             <property name="text">
              <string>Authentication</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>1</width>
                 <height>1</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>26</height>
                </size>
               </property>
               <property name="styleSheet">
                <string>QFrame {
	border: 0px;
	border-top: 2px solid #0099FF;
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="auth_layout">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QCheckBox" name="checkbox_enable_store_auth">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>65</height>
              </size>
             </property>
             <property name="styleSheet">
              <string>QCheckBox {
	color: white;
	font-size: 19pt;
}

QCheckBox::disabled {
	color: grey;
}</string>
             </property>
             <property name="text">
              <string>Store for next sessions</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_reset_auth">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>65</height>
              </size>
             </property>
             <property name="text">
              <string>Reset Current Authentication</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="sync_title_layout">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="sync_icon">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>48</width>
               <height>84</height>
              </size>
             </property>
             <property name="styleSheet">
              <string>padding-bottom: 2px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="title_sync">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string>QLabel#title_download_folder, #title_authentication, #title_sync {
	color: #0099FF;
	font-size: 20pt;
	padding-top: 40px;
	padding-bottom: 10px;
}</string>
             </property>
             <property name="text">
              <string>Synchronizing</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>1</width>
                 <height>1</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QFrame" name="frame_3">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>26</height>
                </size>
               </property>
               <property name="styleSheet">
                <string>QFrame {
	border: 0px;
	border-top: 2px solid #0099FF;
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string>color: white;</string>
           </property>
           <property name="text">
            <string>In order to select sync path you first need to be connected with a DropBox account. Secondly you need to have traveled to the desired path in the content view. The selection tool will be populated with the current sessions metadata. This is the only way to be sure that the sync location is valid.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="sync_buttons_layout">
           <property name="spacing">
            <number>10</number>
           </property>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCheckBox" name="checkbox_only_wlan_sync">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>65</height>
              </size>
             </property>
             <property name="styleSheet">
              <string>QCheckBox {
	color: white;
	font-size: 19pt;
}

QCheckBox::disabled {
	color: grey;
}</string>
             </property>
             <property name="text">
              <string>Only sync while on WLAN</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_sync_now">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>65</height>
              </size>
             </property>
             <property name="text">
              <string>Sync Folder Now</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="sync_layout">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QCheckBox" name="checkbox_enable_sync">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>65</height>
              </size>
             </property>
             <property name="styleSheet">
              <string>QCheckBox {
	color: white;
	font-size: 19pt;
}

QCheckBox::disabled {
	color: grey;
}</string>
             </property>
             <property name="text">
              <string>Enable automated sync</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QFrame" name="sync_frame">
             <property name="styleSheet">
              <string>QFrame#sync_frame {
	background-color: transparent;
	border: 2px solid rgba(255,255,255,150);
	border-radius: 10px;
}

QFrame:disabled#sync_frame {
	border: 2px solid rgba(255,255,255,70);
}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>10</number>
              </property>
              <property name="margin">
               <number>15</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="spacing">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_update_every">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>65</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string>QLabel {
	color: white;
	font-size: 19pt;
}

QLabel::disabled {
	color: grey;
}</string>
                  </property>
                  <property name="text">
                   <string>Sync folder every</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinbox_sync_interval">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>65</height>
                   </size>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="minimum">
                   <number>10</number>
                  </property>
                  <property name="maximum">
                   <number>120</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_min">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>65</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string>QLabel {
	color: white;
	font-size: 19pt;
}

QLabel::disabled {
	color: grey;
}</string>
                  </property>
                  <property name="text">
                   <string>minutes</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttoon_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>69</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>QPushButton#button_cancel, #button_save {
	min-height: 65px;
	border-radius: 0px;
	border: 2px solid rgba(255,255,255,200);
	color: rgb(230, 230, 230);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(16, 16, 16, 255), stop:0.5 rgba(55, 55, 55, 210), stop:1 rgba(36, 36, 36, 255));
	font: Arial;
	font-size: 19pt;
	color: white;
}

QPushButton#button_cancel {
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
	border-right: 0px;
}

QPushButton#button_save {
	border-top-right-radius: 10px;
	border-bottom-right-radius: 10px;
}

QPushButton:pressed#button_cancel, :pressed#button_save {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(59, 59, 59, 255), stop:0.5 rgba(82, 82, 82, 255), stop:1 rgba(68, 68, 68, 255));
	color: #0099FF;
}</string>
       </property>
       <property name="text">
        <string>CANCEL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_save">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>69</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>QPushButton#button_cancel, #button_save {
	min-height: 65px;
	border-radius: 0px;
	border: 2px solid rgba(255,255,255,200);
	color: rgb(230, 230, 230);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(16, 16, 16, 255), stop:0.5 rgba(55, 55, 55, 210), stop:1 rgba(36, 36, 36, 255));
	font: Arial;
	font-size: 19pt;
	color: white;
}

QPushButton#button_cancel {
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
	border-right: 0px;
}

QPushButton#button_save {
	border-top-right-radius: 10px;
	border-bottom-right-radius: 10px;
}

QPushButton:pressed#button_cancel, :pressed#button_save {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(59, 59, 59, 255), stop:0.5 rgba(82, 82, 82, 255), stop:1 rgba(68, 68, 68, 255));
	color: #0099FF;
}</string>
       </property>
       <property name="text">
        <string>SAVE CHANGES</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
